src/sentry/lang/native/minidump.py

```diff
@@ -201,8 +215,27 @@ def get_attached_minidump(data):
 def merge_symbolicator_minidump_response(data, response):
     sdk_info = get_sdk_from_event(data)
 
-    # TODO(markus): Add OS context here when `merge_process_state_event` is no
-    # longer called for symbolicator projects
+    data['platform'] = 'native'
+    data['level'] = 'fatal' if response['crashed'] else 'info'
```

Comment:
In the schema, `crashed` is optional. Even though it's now always set, let's be defensive here. Also, leave the existing value inside.

```py
if response.get('crashed') is not None:
    data['level'] = 'fatal' if response['crashed'] else 'info'
```

Reply:
We can't leave existing values inside (=`setdefault`) unless we remove the `setdefault` in store. iirc there's a setdefault also happening in the store normalizer which makes it more complex.

Reply:
I think I fixed this now.